Makefile-tests.am

```diff
@@ -287,6 +293,17 @@ tests/%-symlink-stamp: % Makefile
 	ln -sf "$${real_bin}" tests/$*; \
 	touch $@
 
+# See above comment on binding the tests to be either installed or not.
+if ENABLE_INSTALLED_TESTS
+installed_test_scripts = $(_installed_or_uninstalled_test_scripts)
+installed_test_programs = $(_installed_or_uninstalled_test_programs)
+check-local:
+	echo "NOTE: Installed tests are enabled; to run them, make install, then: gnome-desktop-testing-runner -p 0 libostree/"
```

Comment:
I understand you're all in on containers, they definitely have appeal. But I doubt I'm the only person who uses `make check`. Otherwise, what was the purpose of #337? I personally would find that maintaining a container infrastructure just so I could make changes to ostree to be an impediment.

Does this change address the issue that some tests can only be run uninstalled? It doesn't seem like it since `make check` becomes a no-op if you build with `--enable-installed-tests` like you're trying to accomplish.

Anyways, just my 2 cents.